src/View/StringTemplate.php

```diff
@@ -178,7 +179,7 @@ protected function _compileTemplates(array $templates = []): void
         foreach ($templates as $name) {
             $template = $this->get($name);
             if ($template === null) {
-                $this->_compiled[$name] = [null, null];
+                throw new InvalidArgumentException(sprintf('String template `%s` is not valid.', $name));
```

Comment:
Should this be "could not be found"?

Reply:
The reason why I named it this way is because we only get in there via the `add()` method.
Therefore `$this->get($name)` will always have a valid key which it retrieves values from but only if the value is `null` we throw the error. See the added test.